tutorials-examples/cmdline-beginner.rst

```diff
@@ -20,6 +20,8 @@
 
 .. This section should provide a brief, top-level description of the page.
 
+**RSP Aspect:** Command-Line
```

Comment:
Now that I see this I realize it should probably be the following, b/c the only aspects are Notebook, Portal, and API. And same for the other command-line tutorial.

**RSP Aspect:** Notebook (terminal command-line)

Reply:
Okay!  I'll change it

Reply:
changed